src/index.js

```diff
@@ -181,99 +181,101 @@ export const Table = ({
   }, [sortBy])
 
   return (
-    <div className={classes.tableMainContainer}>
-      {(_data.length > 0 && toolbar) && (
-        <TableToolbar
-          rows={rows}
-          allColumns={allColumns}
-          visibleColumns={visibleColumns}
-          toggleHideColumn={toggleHideColumn}
-          downloadable={downloadable}
-          data={data}
-          preGlobalFilteredRows={preGlobalFilteredRows}
-          globalFilter={globalFilter || ''}
-          setGlobalFilter={setGlobalFilter}
-          downloadFn={downloadFn}
-        />
-      )}
-      {visibleColumns.length > 0 ? (
-        <TableContainer>
-          <MUITable className={classes.table} {...getTableProps(tableProps)}>
-            <TableHead>
-              {headerGroups.map((headerGroup, i) => (
-                <TableRow key={i} {...headerGroup.getHeaderGroupProps(headerGroupProps)}>
-                  {headerGroup.headers.map((column, i) => (
-                    <TableCell
-                      key={i}
-                      className={classes.head}
-                      {...column.getHeaderProps(column.getSortByToggleProps())}
-                    >
-                      <div className={classes.columnContainer}>
-                        {column.render('Header')}
-                        {column.canSort && (<TableSortLabel {...column} />)}
-                        {column.canFilter && (<TableFilterLabel column={column} />)}
-                      </div>
-                    </TableCell>
-                  ))}
-                </TableRow>
-              ))}
-            </TableHead>
-            <TableBody {...getTableBodyProps()} className={classes.body}>
-              {page.map((row, i) => {
-                prepareRow(row)
-                return (
-                  <TableRow key={i} {...row.getRowProps()}>
-                    {row.cells.map((cell, i) => (
-                      <TableCell key={i} {...cell.getCellProps()}>
-                        {cell.render('Cell')}
+    <ThemeProvider>
```

Comment:
rather than embedding it here, we should relay the responsibility to the applications that use this library, the same way `react-labs` and `lumen-ui` has in the documentation

Reply:
for debugging, testing and adding new features to lumen-ui it is needed so should it be embedded in storybook?

Reply:
Just a heads up, after this change the table will be dependent on the theme provider as lumen UI components do not run without a theme provider.

Reply:
MUI ThemeProvider is only applied locally in overlord so the use of the table in overlord will require a manual wrapping for each component.